e2e/cypress/support/api/role.js

```diff
@@ -136,7 +135,32 @@ export const defaultRolesPermissions = {
         'remove_reaction',
         'manage_slash_commands',
         'get_public_link',
-        'sysconsole_read_environment',
+        'sysconsole_read_environment_web_server',
+        'sysconsole_read_environment_database',
+        'sysconsole_read_environment_elasticsearch',
+        'sysconsole_read_environment_file_storage',
+        'sysconsole_read_environment_image_proxy',
+        'sysconsole_read_environment_smtp',
+        'sysconsole_read_environment_push_notification_server',
+        'sysconsole_read_environment_high_availability',
+        'sysconsole_read_environment_rate_limiting',
+        'sysconsole_read_environment_logging',
+        'sysconsole_read_environment_session_lengths',
+        'sysconsole_read_environment_performance_monitoring',
+        'sysconsole_read_environment_developer',
+        'sysconsole_write_environment_web_server',
+        'sysconsole_write_environment_database',
+        'sysconsole_write_environment_elasticsearch',
+        'sysconsole_write_environment_file_storage',
+        'sysconsole_write_environment_image_proxy',
+        'sysconsole_write_environment_smtp',
+        'sysconsole_write_environment_push_notification_server',
+        'sysconsole_write_environment_high_availability',
+        'sysconsole_write_environment_rate_limiting',
+        'sysconsole_write_environment_logging',
+        'sysconsole_write_environment_session_lengths',
+        'sysconsole_write_environment_performance_monitoring',
+        'sysconsole_write_environment_developer',
```

Comment:
Make sure you're also giving any ancillary permissions they should be getting with these